ipamd/ipamd.go

```diff
@@ -664,3 +700,35 @@ func (c *IPAMContext) eniIPPoolReconcile(ipPool map[string]*datastore.AddressInf
 	return nil
 
 }
+
+func getWarmIPTarget() int {
+	inputStr, found := os.LookupEnv("WARM_IP_TARGET")
```

Comment:
Instead of `WARM_IP_TARGET` which I don't think is very descriptive, I propose `NUM_WARM_IPS`, or `MAX_WARM_IPS` each of which signifies a number rather than a resource.

Reply:
Out of curiosity how is this supposed to work if you assigned a value of 5? Right from the start there will be 5 secondary IPs available for pods to use, but let's say 3 pods are started on that particular machine... Does the pool of secondary IPs get expanded back out to 5, if so does it allocate three new addresses one at a time? Alternatively does the pool of available IPs become 2, then when those 2 IPs are eventually used then a new block of 5 IPs will be attached when needed. The naming for the former might be `NUM_WARM_IPS` while the later might be `IP_ALLOCATION_BLOCK_COUNT` or something along those lines.

Reply:
if `WARM_IP_TARGET` is 5,
* at beginning, the available IPs is 5
* after 3 Pods assigned,  ipamD will try to allocate 3 more IPs  in the ipamD controller loops. ipamD will try to use 1 EC2 API if it is possible.  If it requires allocating 1 more ENIs, or EC2 control plane can not allocate 3 IP address at that time, ipamD controller loops will continuously retry to reach the `target` of 5 IPs

There are 2 reasons I choose WARM_IP_TARGET

* There is another configure knob `WARM_ENI_TARGET` which works similar but at ENI level.
  * the default is 1, e.g. if it is configured at 2, ipamD will try to make 2 ENI available. PR #68

* `...TARGET` means the `desired` value, and ipamD controller loops will try continuously to reach this `TARGET`.

Reply:
Ok, I see `ipPoolMonitorInterval` is set to run every 5 seconds to check to see if it needs to increase the IP pool. Is that value perhaps set too low? Or if many pods are launched simultaneously will it all fall within the same time window to not cause excessive EC2 API calls.